### PR TITLE
[COM-248] Adds prettier to middleware registry

### PR DESCRIPTION
**Pull request by hannahtrask**
- Adds prettier
- Matches other TryNow NextJS prettier configuration
- Handles eslint overrides

### CODE REVIEW

**Review by alxgmpr — APPROVED** [review]
Looks good to me, thanks for doing this. Should probably wait until the current open PRs are merged down before fixing conflicts.

**Review by joshrathke — APPROVED** [review]

**Pull request merged**
